tests/test_stats/features/features_interpolation.json

```diff
@@ -2,35 +2,58 @@
   "eopatch-id-0-col-0-row-0": {
     "bbox": "BBox(((729480.0, 4390045.0), (732120.0, 4391255.0)), crs=CRS('32638'))",
     "data": {
-      "FEATURES": [
-        23,
-        121,
-        264,
-        16,
-        "float64",
-        1022208,
-        10733184,
-        -0.68999,
-        1.01031,
-        0.12296,
-        0.1381,
-        70357,
-        360698,
-        527532,
-        5750765,
-        3691296,
-        271950,
-        60530,
-        56,
-        0.07232,
-        0.16427,
-        0.15619,
-        0.13258,
-        0.17291,
-        0.13551,
-        0.06184,
-        0.10461
-      ]
+      "FEATURES": {
+        "array_shape": [
+          23,
+          121,
+          264,
+          16
+        ],
+        "basic_stats": {
+          "max": 1.01031,
+          "mean": 0.12296,
+          "median": 0.1381,
+          "min": -0.68999
+        },
+        "counts": {
+          "infinite": 0,
+          "nan": 1022208
+        },
+        "dtype": "float64",
+        "histogram": {
+          "counts": [
+            70357,
+            360698,
+            527532,
+            5750765,
+            3691296,
+            271950,
+            60530,
+            56
+          ],
+          "edges": [
+            -0.68999,
+            -0.47746,
+            -0.26492,
+            -0.05238,
+            0.16016,
+            0.3727,
+            0.58524,
+            0.79777,
+            1.01031
+          ]
+        },
+        "random_values": [
+          0.0789,
```

Comment:
This is intentional. It is because now we sample from the whole numpy array while before we sampled from a reduced array that was missing nans and infinite values. IMO, it is better to sample from the whole array.

(I will resolve all other threads that ask the same question.)